pipelines/timeseries_forecasts.R

```diff
@@ -68,9 +71,9 @@ fit_and_forecast <- function(data,
                              n_samples = 2000,
                              target_col = "ed_visits",
                              output_col = "other_ed_visits") {
-  forecast_horizon <- glue::glue("{n_forecast_days} days")
-  target_sym <- rlang::sym(target_col)
-  output_sym <- rlang::sym(output_col)
+  forecast_horizon <- glue("{n_forecast_days} days")
+  target_sym <- sym(target_col)
+  output_sym <- sym(output_col)
```

Comment:
At some point we should make this more consistent with https://dplyr.tidyverse.org/articles/programming.html but it doesn't have to be now.

Reply:
https://github.com/CDCgov/pyrenew-hew/issues/150